ql/src/java/org/apache/hadoop/hive/ql/exec/DDLTask.java

```diff
@@ -1159,6 +1160,12 @@ private int renamePartition(Hive db, RenamePartitionDesc renamePartitionDesc) th
       return 0;
     }
 
+    String names[] = Utilities.getDbTableName(tableName);
```

Comment:
this will be related to the distcp failure though when copying data and concurrent exchange of partition.